### PR TITLE
DEVDOCS-4399: [net new] Tax Settings

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4399](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4399)

Related to:
-[PR 946](https://github.com/bigcommerce/api-specs/pull/946) (api spec)
-[PR 132](https://github.com/bigcommerce-labs/next-dev-center-prototype/pull/132) (redirects) 

ping @bc-ani

### CODE REVIEW

File: docs/api-docs/tax/tax-settings.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Tax Settings
+
+Tax settings allow you to choose how a store handles tax calculations and price display. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.md:3]:
```suggestion
Tax settings allow you to choose how a store handles tax calculations and price displays. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```

File: docs/api-docs/tax/tax-settings.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+This guide shows you how to use the Tax Settings API. For more info, see the [Tax Settings API Reference](/api-reference/store-management/tax-settings). 
+
+## Get tax settings
+
+To get tax settings, send a request to the [Get tax settings](/api-reference/store-management/tax-settings/tax-settings/get-tax-settings) endpoint
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.md:9]:
```suggestion
To get tax settings, send a request to the [Get tax settings](/api-reference/store-management/tax-settings/tax-settings/get-tax-settings) endpoint.
```

File: docs/api-docs/tax/tax-settings.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  },
+  "meta": {}
+}
+```
+
+<!-- type: tab-end -->
+
+## Update tax settings
+
+To update tax settings, send a request to the [Update tax settings](/api-reference/store-management/tax-settings/tax-settings/update-tax-settings) endpoint. The request updates only fields that you specify.
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.md:46]:
```suggestion
To update tax settings, send a request to the [Update tax settings](/api-reference/store-management/tax-settings/tax-settings/update-tax-settings) endpoint. The request updates only the fields that you specify.
```

**Review by bc-traciporter — COMMENTED** [review]
I made some editorial updates.

**Review by bc-traciporter — APPROVED** [review]

File: docs/api-docs/tax/tax-settings.md (hunk 1 above)
Comment (theromulans) [reply docs/api-docs/tax/tax-settings.md]: Hmm I'm not sure I'd agree with pluralising `display`.

File: docs/api-docs/tax/tax-settings.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+# Tax Settings
+
+Tax settings allow you to choose how a store handles tax calculations and price displays. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```
Comment (theromulans) [inline docs/api-docs/tax/tax-settings.md:3]: > You can also choose whether prices include taxes when displayed on storefront orders.

This isn't quite accurate - in particular the usage of _storefront_ is bothersome. The settings available today speak to the control panel view and invoices. A shopper can access invoices from their account > order list.

If anything I'd remove the word "storefront" - this should hopefully avoid misleading people. For further context, the primary function of storefront price display is handled via _tax zones_.

Comment (theromulans) [inline docs/api-docs/tax/tax-settings.md:3]: > Tax settings allow you to choose how a store handles tax calculations and price displays.

I'd go the reverse angle and remove the pluralisation here.

Suggestion:

> Tax settings allow you to choose how a store handles tax calculation and price display.

It is worth noting that we plan to expand this endpoint to do more things in the future, beyond just these two. I suppose we can add a catch all when that day comes?

**Comment by bc-andreadao** [comment]
@theromulans Thanks for the help. I applied the wording

**Review by theromulans — APPROVED** [review]
Thanks again @bc-andreadao.

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**